mantidproject/mantid
# Add Workspace to Scale Option to the ISIS Reflectometry GUI

Fixes #14846

**Tester**

Code review the changes, then set about exercising the functionality via the MantidPlot ->Interface->Reflectometry->ISIS [Reflectometry GUI](http://www.mantidproject.org/ISIS_Reflectometry_GUI)

To do this, get the files listed here http://www.mantidproject.org/Testing_ISIS_Reflectometry_GUI#Requirements

Checking the Menu Options
1. Open the UI
2. Go to Options->Refl GUI Options
3. The option _Scale Right Workspace When Stitching_ should be checked
4. Close the options

Run the UI in the default mode
1.  Open the table (*.tbl file listed above) via the File->Open Table Option.
2.  Select the first row in the table. Select the 'Stitch' checkbox on that row.
3. Hit 'Process'
4. You'll get a workspace called '13460_62' created. Rename it to  '13460_62_left'

Change the Options and Re-run
1. Now change the option _Scale Right Workspace When Stitching_  to be deslected.
2. Rerun the same row in the same way as above
3. Rename the output Rename the output '13460_62' to '13460_62_left'
4. Plot both and you should get something like this.
   ![screen shot 2015-12-23 at 10 39 38](https://cloud.githubusercontent.com/assets/1109536/11975190/1806a8c2-a963-11e5-8f6e-690f9ac09601.png)

TODO:
Release notes and online documentation for this interface will need updating
